laplace/curvature/backpack.py

```diff
@@ -41,7 +41,7 @@ def jacobians(self, x):
                 if model.output_size > 1:
                     out[:, i].sum().backward()
                 else:
-                    out.sum().backward()
+                    out.sum().backward(retain_graph=True)
```

Comment:
Here potentially `create_graph=True` if pred uct. needs to be differentiable. We should, at least in the future, consider this to be a backend option to turn on and off since it costs more memory to have these set to true.

Reply:
Fixed [bc50025](https://github.com/AlexImmer/Laplace/pull/120/commits/bc50025d079abae01a65ed1dce8c3e3fe0975e6b)